app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class UserSerializer < ActiveModel::Serializer
```

Comment:
Do we need this one?

Reply:
Yes, because in the apiary body they request the user. https://wbooksapi.docs.apiary.io/#reference/users-rents/collection/create

Reply:
Cool! Usually the Apiary is a contract between the API team and the Frontend/mobile team to establish the interface to work with. So, it's good to have this serializer. Nevertheless, pay attention that not all attributes match the ones in the Apiary. For now it's okay (you don't have to change it) but bear in mind.